data/remnant missions.txt

```diff
@@ -495,3 +565,691 @@ mission "Remnant: Technology Available"
+		not "Remnant: Plasma Cannons: done"
+		not "Remnant: Heavy Laser: active"
+		not "Remnant: Heavy Laser: done"
+		random < 50
+	on offer
+		require "Catalytic Ramscoop"
+		conversation
+			`As you are walking through the spaceport, a woman comes up to you and introduces herself as an engineer at one of the Remnant development labs. She pauses awkwardly before beginning to chant: "As your ship is the only one we have seen in many decades that has outfits from ancestral space, we are intrigued by it. Could we take a closer look?"`
+			choice
+				`	"Yes, you are welcome to examine it."`
+					goto deep
+				`	"Yes, but don't touch anything."`
+				`	"No, stay away from my ship."`
+					decline
+
+			`	"I'll be careful," she sings with a tinge of sadness. "I am an engineer, not a Korath."`
+				goto more
+
+			label deep
+			`	"Fascinating," she sings as she clambers over <ship> with the grace of someone who obviously knows her ships. "Above all else, I am an engineer of movement," she chants. "We have done well to hide as deep in the Ember Waste as we have, but our ships' limited ranges struggle to take us farther. Examining systems from elsewhere may inspire new developments."`
+
+			label more
+			`	After examining <ship> for quite some time, she stops beneath it and looks up. "Is this a new form of ramscoop?" she asks as she gestures at your ship's ramscoop. As you nod in the affirmative, she continues. "Fuel has always been a great limiter of our exploration. If you could get a couple extra that we could dismantle, this could be very useful to us."`
+			choice
+				`	"Sure, I could bring you some."`
+				`	"Sorry, I'm really not interested in helping Remnant research."`
+					decline
+
+			`	After a short discussion, she agrees to pay you <payment> in exchange for a cargo of two catalytic ramscoops.`
+				accept
+	
+	on complete
+		outfit "Catalytic Ramscoop" -2
+		payment 1390000
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program.`
+		conversation
+			`You visit the local shipyard and eventually find the engineer busily repairing a damaged Starling. You inform her that you have brought a pair of catalytic ramscoops for her to study. She quickly tightens a couple more connectors and jumps down from her perch, gesturing at an assistant to transfer you the credits you were promised. "Thank you! Our maps of human space are old, and if they are still accurate, you traveled a very long way to bring these to us." Her chanting voice echoes off the Starling above you with an interesting resonance.`
+			`	"If we can unlock the secrets of these ramscoops, it will give our ships greater range as they search for a safer refuge." As she picks up the remote control for the small loading truck with the two ramscoops she trills at you over her shoulder. "Oh, my name is Tali. I'll let you know what I find."`
```

Comment:
Well, much as they might like the constant stream of salvage the Korath provide, there's always the risk that the Korath decide "enough is enough" and really bring down the hammer on them and send a warfleet instead of a minor pillaging force. There's also the fact that if one group could find them, then chances are the Alphas and other hostiles can do.

Reply:
fair enough, that's a subjective question anyways

Reply:
Yeah. They're a paranoid group. I'm not sure *anywhere* would be good enough refuge for them.